### PR TITLE
📖 Update Try-it page based on the v1beta1 API

### PR DESCRIPTION
**Pull request by ren-yifei**
Update the Try-it.md to include the changes from release v1beta1.
The session 2.5 Running custom CAPM3 is not finished, since the
make run command is deprecated.

### CODE REVIEW

File: try-it.md
Comment (mboukhalfa) [inline try-it.md:291]: should be fixed

File: try-it.md
Comment (Xenwar) [inline try-it.md:523]:
```suggestion
| b423ee9c-66d8-48dd-bd6f-656b93140504 | node-1 | 7f51f14b-7701-436a-85ba-7dbc7315b3cb | power off   | available          | False       |
| 882533c5-2f14-49f6-aa44-517e1e404fd8 | node-0 | 14fbcd25-4d09-4aca-9628-a789ba3e175c | power off   | available          | False       |
```

Comment (Xenwar) [reply try-it.md:523]: not sure if there is a change to the naming of bmhs

File: try-it.md
Comment (lentzi90) [inline try-it.md:494]: This sounds strange. Is it when using Tilt? In that case I think it should be mentioned. E.g. "... changes will be automatically applied when using Tilt."

File: try-it.md
Comment (lentzi90) [inline try-it.md:39]: I think this is not necessary but please correct me if I'm wrong. It should be possible to run all of this locally without any floating IP.

File: try-it.md
Comment (lentzi90) [inline try-it.md:486]: No big deal, but when including the `$` prompt, it would be more correct to use `console` or `shell` instead of `sh`. I noticed that this guide is using `sh` all the time and I think it would be nice to change it to `console`. It makes the syntax highlighting correct.

See https://stackoverflow.com/a/49004070 .

**Review by lentzi90 — COMMENTED** [review]
Good to see this updated!

**Review by lentzi90 — APPROVED** [review]
Great job!

Edit: remember to squash the commits before merging!

File: try-it.md
Comment (fmuyassarov) [inline try-it.md:293]:
```suggestion
**CAPM3_VERSION** set to **v1beta1**. This is the default behavior. The v1beta1 deployment can be done with
Ubuntu 20.04 or Centos 8 target host images. Please make sure to meet [resource requirements](#11-prerequisites) for successful deployment:
```

File: try-it.md
Comment (fmuyassarov) [inline try-it.md:493]: we haven't talked about Tilt until this point and not sure if the mention of it will not confuse people.
I think either we need to explain how to work with Tilt/give a reference to a document where we have explained or
keep this part as it is

**Comment by fmuyassarov** [comment]
Also I would keep code blocks as shell format instead of console. They look a bit different in the preview website: https://deploy-preview-224--metal3.netlify.app/try-it.html

**Comment by lentzi90** [comment]
> Also I would keep code blocks as shell format instead of console. They look a bit different in the preview website: https://deploy-preview-224--metal3.netlify.app/try-it.html

But shell gives correct syntax highlighting when the prompt is included. Then it knows what is output (no syntax highlighting) and what is the command (with syntax highlighting).
There is one part of the page where `console` is used wrong though: https://github.com/metal3-io/metal3-io.github.io/blob/f1fcf31f378814901ae622ec829032e31c0d3f23/try-it.md?plain=1#L434-L451
Here the comments (starting with `#`) are treated as root prompt. (`$` is normal user prompt, `#` is root). If we want these highlighted as comments they should have the prompt in front (`$ # Comment here`) or we switch that block to shell.

I vote for putting the prompt before the comment. Then it is obvious what part is output and what is input. :slightly_smiling_face:

**Comment by ren-yifei** [comment]
I updated the code blocks part using console, and improved the comments lines in the block, please check whether that looks better or not.
Also I added a brief introduction part for Tilt, please check that all the information is correct or not.

**Review by fmuyassarov — COMMENTED** [review]
/lgtm
/cc @hroyrh

**Comment by fmuyassarov** [comment]
/hold
@ren-yifei can you please squash commits before we merge it?

File: try-it.md
Comment (furkatgofurov7) [inline try-it.md:499]:
```suggestion
Tilt is applied in the metal3-dev-env deployment, you can make changes inside the
`cluster-api-provider-metal3` folder and Tilt will deploy the changes automatically.
```

File: try-it.md
Comment (furkatgofurov7) [inline try-it.md:35]:
```suggestion
- System with Centos 8 Stream or Ubuntu 20.04
```

File: try-it.md
Comment (furkatgofurov7) [inline try-it.md:39]: I would suggest not to include these as dependencies since most of them will be installed by m3-dev-env scripts

File: try-it.md
Comment (furkatgofurov7) [inline try-it.md:293]:
```suggestion
Ubuntu 20.04 or Centos 8 Stream target host images. Please make sure to meet [resource requirements](#11-prerequisites) for successful deployment:
```

File: try-it.md
Comment (furkatgofurov7) [inline try-it.md:500]:
```suggestion
If you deployed CAPM3 separately and want to make changes to it, then
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
Overall lgtm, small nit that needs a fix:

**Comment by furkatgofurov7** [comment]
/retitle 📖 Update Try-it page based on the v1beta1 API

**Review by furkatgofurov7 — COMMENTED** [review]
LGTM

**Pull request merged**
